voltdbclient/rows.go

```diff
@@ -496,3 +496,16 @@ func bytesToTime(bs []byte) time.Time {
 	// time.Unix will take either seconds or nanos. Multiply by 1000 and use nanos.
 	return time.Unix(0, millis*1000)
 }
+
+// HasNextResultSet implements driver.RowsNextResultSet
+func (vr *VoltRows) HasNextResultSet() bool {
+	return vr.tableIndex+1 < vr.getNumTables()
```

Comment:
Should we also make sure table is valid (vr.tableIndex != -1)?

Reply:
if that is a possibility then yes...